main.go

```diff
@@ -5,21 +5,23 @@
 package main
 
 import (
+	"errors"
 	"net/http"
+	"os"
 
 	"github.com/drone/drone-convert-starlark/plugin"
 	"github.com/drone/drone-go/plugin/converter"
-
 	_ "github.com/joho/godotenv/autoload"
 	"github.com/kelseyhightower/envconfig"
 	"github.com/sirupsen/logrus"
 )
 
 // spec provides the plugin settings.
 type spec struct {
-	Bind   string `envconfig:"DRONE_BIND"`
-	Debug  bool   `envconfig:"DRONE_DEBUG"`
-	Secret string `envconfig:"DRONE_SECRET"`
+	Bind         string `envconfig:"DRONE_BIND"`
+	Debug        bool   `envconfig:"DRONE_DEBUG"`
+	Secret       string `envconfig:"DRONE_SECRET"`
+	LoadRootPath string `envconfig:"DRONE_LOAD_ROOT_PATH"`
```

Comment:
let's call this something like `DRONE_LIBRARY_PATH` or `_MODULE_PATH` or something that is just a bit more descriptive.